code-challenges.test.js

```diff
@@ -16,30 +18,102 @@
 // --------------------1) Create a function that takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
 
 // a) Create a test with an expect statement using the variable provided.
+//set up test that will describe when codedMessage gets called
+// if argument is secretCodeWord1 
+//actual result .toEqual("l4ck4d41s1c4l")
+// if argument is secretCodeWord2 
+//actual result .toEqual("g0bbl3dyg00k")
+
+describe('when codedMessage gets called',()=>{
+    it("if argument is secretCodeWord1, retrun l4ck4d41s1c4l",() =>{
```

Comment:
try to genericize the description so that it describe behavior rather than explicit inputs and outputs. Generally descriptions shouldn't know about code